cpp/src/barretenberg/honk/proof_system/verifier.hpp

```diff
@@ -20,9 +20,9 @@ template <typename Flavor> class StandardVerifier_ {
     bool verify_proof(const plonk::proof& proof);
 
     std::shared_ptr<VerificationKey> key;
-    std::map<std::string, Commitment> kate_g1_elements;
-    std::map<std::string, FF> kate_fr_elements;
-    std::shared_ptr<PCSVerificationKey> kate_verification_key;
+    std::map<std::string, Commitment> pcs_g1_elements;
```

Comment:
If these are just commitments, maybe we should rename it to something like that? The g1_elements only has meaning in parining-based commitments